### DISCUSSION TITLE
posts/blog-dilemma

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#10] [General]
# posts/blog-dilemma/

a student obsessed with the computing world

https://www.whexy.com/posts/blog-dilemma/

**Comment by MstMoonshine** [comment]
Good point. It reminds me of [another blog](https://www.dingmos.com/) subscribed by me. Most posts in the blog are quite short, nevertheless helpful very often. And I personally appreciate this style.
